src/main/java/core/basesyntax/figure/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.color.Color;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final String FIGURE_NAME = "isosceles trapezoid";
```

Comment:
```suggestion
```
You don't need this constant, just enter the name of the figure by hand. Fix everywhere